File: Ticktalk/app/src/main/java/com/example/ticktalk/MainActivity.java
package com.example.ticktalk;
import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;

import android.content.DialogInterface;
import android.content.Intent;
import android.graphics.Color;
import android.os.Bundle;
import android.widget.LinearLayout;

import com.google.android.gms.tasks.OnCompleteListener;
import com.google.android.gms.tasks.Task;
import com.google.firebase.remoteconfig.FirebaseRemoteConfig;
import com.google.firebase.remoteconfig.FirebaseRemoteConfigSettings;

import static androidx.appcompat.app.AlertDialog.*;


public class MainActivity extends AppCompatActivity {

    private static final String TAG = "MyTag";

    private LinearLayout linearLayout;
    private FirebaseRemoteConfig mFirebaseRemoteConfig;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        linearLayout = findViewById(R.id.mainactivity_linearlayout);

        mFirebaseRemoteConfig = FirebaseRemoteConfig.getInstance();
        FirebaseRemoteConfigSettings configSettings = new FirebaseRemoteConfigSettings.Builder()
                .setDeveloperModeEnabled(BuildConfig.DEBUG)
                .setMinimumFetchIntervalInSeconds(0)
                .build();
        mFirebaseRemoteConfig.setConfigSettingsAsync(configSettings);
        mFirebaseRemoteConfig.setDefaultsAsync(R.xml.remote_config_defaults);

        mFirebaseRemoteConfig.fetchAndActivate()
                .addOnCompleteListener(this, new OnCompleteListener<Boolean>() {
                    @Override
                    public void onComplete(@NonNull Task<Boolean> task) {
                        if (task.isSuccessful()) {

                        } else {
                        }
                        displayMessage();
                    }
                });

    }

    void displayMessage() {

        String main_background = mFirebaseRemoteConfig.getString("main_background");
        boolean caps = mFirebaseRemoteConfig.getBoolean("main_message_caps");
        final String main_message = mFirebaseRemoteConfig.getString("main_message");

        linearLayout.setBackgroundColor(Color.parseColor(main_background));

        if (caps) {
            Builder builder = new Builder(this);
            builder.setMessage(main_message);
            builder.setPositiveButton("확인", new DialogInterface.OnClickListener() {
                @Override
                public void onClick(DialogInterface dialog, int which) {
                    finish();
                }
            });

            builder.create().show();
        } else{
        }

        startActivity(new Intent(this,LoginActivity.class));

    }
}

File: todolist/app/src/main/java/com/example/todolist/MainActivity.kt
package com.example.todolist

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val recyclerView = findViewById<RecyclerView>(R.id.recyclerView)
        recyclerView.layoutManager = LinearLayoutManager(this, LinearLayoutManager.VERTICAL, false)

        val mainWork = ArrayList<Work>()

        var getWork = intent.getStringExtra("work")
        var getPeriod = intent.getStringExtra("date")


        if (getWork == null || getPeriod == null) {
            println("")
        } else {
            mainWork.add(Work(getWork, getPeriod))
            var adapter = Adapter(mainWork)
            recyclerView.adapter = adapter

        }
    }

    fun changeActivity(view: View) {
        val intent = Intent(this@MainActivity, AddWork::class.java)
        startActivity(intent)
    }


}

